boards/shared/Inc/App/App_SharedError.h

```diff
@@ -0,0 +1,139 @@
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+
+struct Error;
+struct ErrorList;
+struct ErrorBoardList;
+
+enum ErrorBoard
+{
+    BMS,
+    DCM,
+    DIM,
+    FSM,
+    PDM,
+    NUM_ERROR_BOARDS,
+};
```

Comment:
Perhaps it's worth creating `enum Board` in a separate file so that we can share this with other code in the future?

Reply:
Yeah I was thinking the same too, I will create one under `shared`

Reply:
Fixed